__tests__/unit/lib/utils/repoGitDiff.test.js

```diff
@@ -52,6 +53,20 @@ describe(`test if repoGitDiff`, () => {
     expect(work).toStrictEqual(output)
   })
 
+  test('can filter ignored files', () => {
+    const output = [
+      'M      force-app/main/default/lwc/jsconfig.json'
+    ]
+    child_process.spawnSync.mockImplementation(() => ({
+      stdout: output[0]
+    }))
+    // eslint-disable-next-line no-undef
+    const work = repoGitDiff({ output: '', repo: '', ignore: '__tests__/unit/lib/utils/.forceignore' }, globalMetadata)
```

Comment:
use a constant at the beginning of the file for the path to the forceignore file plz